clearnode/export_transactions_cli_test.go

```diff
@@ -0,0 +1,165 @@
+package main
+
+import (
+	"bytes"
+	"encoding/csv"
+	"testing"
+
+	"github.com/shopspring/decimal"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTransactionExporter_ExportToCSV(t *testing.T) {
+	db, cleanup := setupTestDB(t)
+	t.Cleanup(cleanup)
+
+	logger := NewLoggerIPFS("test")
+	exporter := NewTransactionExporter(db, logger)
+
+	// Create test data
+	account1 := "0x1234567890123456789012345678901234567890"
+	account2 := "0x0987654321098765432109876543210987654321"
+	account3 := "app-session-123"
+
+	// Create user tags for wallet accounts
+	tag1, err := GenerateOrRetrieveUserTag(db, account1)
+	require.NoError(t, err)
+	tag2, err := GenerateOrRetrieveUserTag(db, account2)
+	require.NoError(t, err)
+
+	// Create test transactions
+	_, err = RecordLedgerTransaction(db, TransactionTypeTransfer, NewAccountID(account1), NewAccountID(account2), "usdc", decimal.NewFromInt(100))
+	require.NoError(t, err)
+
+	_, err = RecordLedgerTransaction(db, TransactionTypeDeposit, NewAccountID(account2), NewAccountID(account1), "eth", decimal.NewFromInt(50))
+	require.NoError(t, err)
+
+	_, err = RecordLedgerTransaction(db, TransactionTypeAppDeposit, NewAccountID(account1), NewAccountID(account3), "usdc", decimal.NewFromInt(25))
+	require.NoError(t, err)
+
+	t.Run("Export", func(t *testing.T) {
```

Comment:
How about try to run the test cases in parallel?

```suggestion
	t.Parallel()
	
	t.Run("Export", func(t *testing.T) {
```

If something blows up, we may want to take a look at how the fixtures are initialized then (and quite possibly do so in the context of a separate work item/follow-up PR)

Reply:
updated https://github.com/erc7824/nitrolite/pull/256/commits/a6c4b23c3c3d0bb25c68ce1d073b4758597844ed